Samples/WinMLSamplesGallery/WinMLSamplesGallery/Samples/DXResourceBindingORT/DXResourceBindingORT.xaml.cs

```diff
@@ -0,0 +1,108 @@
+using Microsoft.AI.MachineLearning;
+using Microsoft.UI.Xaml;
+using Microsoft.UI.Xaml.Controls;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Threading.Tasks;
+using Windows.Graphics.Imaging;
+using Windows.Media;
+using Windows.Storage;
+using WinMLSamplesGalleryNative;
+using System.Linq;
+
+namespace WinMLSamplesGallery.Samples
+{
+    public class DXClassifierResult
+    {
+        public int index { get; private set; }
+        public string label { get; private set; }
+        public DXClassifierResult(int i, string lab)
+        {
+            index = i;
+            label = lab;
+        }
+    }
+
+    public sealed partial class DXResourceBindingORT : Page
+    {
+        private bool pageExited = false;
+        public DXResourceBindingORT()
+        {
+            this.InitializeComponent();
+        }
+
+        private async void LaunchWindow(object sender, RoutedEventArgs e)
+        {
+            ToggleLaunchWindowButton();
+            ToggleLoadingTxtVisibility();
+
+            await Task.Run(() => WinMLSamplesGalleryNative.DXResourceBinding.LaunchWindow());
+
+            ToggleLoadingTxtVisibility();
+            ClassifyCube();
+        }
+
+        private async void ClassifyCube()
+        {
+            int i = 0;
+            float[] prev_results = { };
+            while (true)
+            {
+                if (pageExited)
+                {
+                    WinMLSamplesGalleryNative.DXResourceBinding.CloseWindow();
+                    break;
+                }
+                float[] results = await Task.Run(() => WinMLSamplesGalleryNative.DXResourceBinding.EvalORT());
+                if (i == 0)
+                    prev_results = results;
+                // The first evaluation may return null so move to the next iteration
+                if (results == null)
+                    continue;
+                UpdateClassificationResults(prev_results);
+                prev_results = results;
+                System.Threading.Thread.Sleep(1000);
```

Comment:
Since this is already an `async` function, should this be an `await` statement rather than a sleep? Direct sleeps are bad :b.